server.go

```diff
@@ -313,20 +314,20 @@ type peerState struct {
 }
 
 // ConnectionsWithIP returns the number of connections with the given IP.
-func (ps *peerState) ConnectionsWithIP(ip net.IP) int {
+func (ps *peerState) ConnectionsWithIP(ip []byte) int {
```

Comment:
Perhaps this is being changed for an upcoming commit, but it doesn't really make sense to me to make this change in isolation as of this commit.

Would you mind elaborating on the thought process for the change?